src/config.ts

```diff
@@ -216,13 +216,17 @@ function getProvider(
   config: Record<string, any>,
   metamask: ethers.providers.JsonRpcProvider | null,
 ): ethers.providers.JsonRpcProvider {
-  // Use Alchemy in production. Otherwise use Metamask if installed.
   if (import.meta.env.PROD) {
     return new ethers.providers.AlchemyWebSocketProvider(network.name, config.alchemy.key);
   } else if (metamask) {
     return metamask;
+  } else if (import.meta.env.DEV) {
+    // The ethers defaultProvider doesn't include a `send` method, which breaks the `utils.getTokens` fn.
+    // Since Metamask nor WalletConnect provide an `alchemy_getTokenBalances` nor `alchemy_getTokenMetadata` endpoint we can rely on not using `config.provider.send`.
+    // TODO: Improve the provider used for local usage
```

Comment:
What's the TODO about? by local usage you mean DEV?

Reply:
The TODO is about that we eventually should find a way to use a `JsonRpcProvider`, since we eventually we'll find issues with using a `BasicProvider` casted as `JsonRpcProvider`